rfcs/0001-incentivization-transactions.md

```diff
@@ -0,0 +1,66 @@
+# Summary
+
+[summary]: #summary
+
+Provide a mechanism to pay users for having a verified identity and continuously adding data to their data host.
+
+# Motivation
+
+[motivation]: #motivation
+
+The protocol is deeply dependant on the network effects of users wanting applications and applications wanting users.
+To grow faster, we want to directly incentivize users to be on the network and create data.
+
+We expect users to proactively set up their identity, data capture, and data hosting to receive FCL tokens.
+
+# Guide explanation
+
+[guide-explanation]: #guide-explanation
+
+You can receive a portion of FCL minting by:
+
+1. Hosting a signed Fractal ID message.
+1. Regularly publishing the Merkle root of all data in your data host.
+
+# Reference explanation
+
+[reference-explanation]: #reference-explanation
+
+Mint tokens for addresses that publish the following:
+
+1. A unique ID signed by Fractal ID.
+1. A Merkle root containing more data than the previously published one for that Fractal ID.
+
+We will mint tokens with the following constraints:
+
+1. Constant amount per user per day, when below total issuance per day.
+   This ensures users are indifferent towards more users using the system.
+1. Maximum total issued per day, with minting evenly distributed among IDs.
+
+Additionally, communicate to users that we will retroactively be incentivizing genuine data.
+Users should have these Merkle roots include their genuine browsing data, as they will be paid for that later.
```

Comment:
Does this mean "if you provide data at t0 and only hold a signed ID at t1, we'll pay you back the incentives from t0"?

Reply:
It means that once we have the infrastructure to analyze data for genuineness and mint based on that, we will then mint somehow including data from this earlier phase of minting. So capturing genuine data now will get you paid more later than if you just started capturing genuine data later.

Reply:
So a scenario would be that we distribute incentives to those with a signed ID and data, AND promise users that later, when we deploy additional mechanisms to ascertain genuineness, those will apply retroactively?

Reply:
I believe so. I think we want to maximize the amount of genuine data that we capture before we have the infrastructure to analyze it. 

It would be irrelevant if we launch the mainnet with the ability to incentivize based on data genuineness. In that case, we don't have this incentivization phase, so no reason to make promises for it.

Reply:
Sounds good to leave the option open, but let's take care to avoid setting expectations we don't want to break :)

Reply:
Done.